cmd/fluent-watcher/fluentbit/main.go

```diff
@@ -187,6 +189,9 @@ func start() {
 	}
 
 	cmd = exec.Command(binPath, "-c", configPath)
+	if externalPluginPath != "" {
+		cmd = exec.Command(binPath, "-e", externalPluginPath)
```

Comment:
I think ` "-c", configPath` is still needed here?

Reply:
Yes, you are right.
Fixed